### PR TITLE
[lighthouse] add doc for progressive enhancement audit

### PR DESCRIPTION
**Pull request by kaycebasques**
@addyosmani WDYT?

### CODE REVIEW

File: src/content/en/tools/lighthouse/no-js.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+{# wf_updated_on: 2016-09-20 #}
+{# wf_published_on: 2016-09-20 #}
+
+# Page Contains Some Content When Its Scripts Are Not Available  {: .page-title }
+
+## Why the audit is important {: #why }
+
+[Progressive Enhancement](https://en.wikipedia.org/wiki/Progressive_enhancement)
+is a web development strategy that ensures that your site is accessible to the
+greatest possible audience. The most common definition of progressive
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/no-js.md]: largest instead of greatest?

**Comment by petele** [comment]
@kaycebasques - this is going into `lighthouse` but your other PR went into `next-version`, will this get merged into next-version?

**Comment by kaycebasques** [comment]
@petele whoops, I meant for that other PR to merge into `lighthouse`, too

`lighthouse` is branched off of `next-version`, I'm PRing the docs in smaller batches. I need to get technical reviews on them. When all of the docs are ready, I'll merge `lighthouse` into `next-version`

**Comment by PaulKinlan** [comment]
@petele outside of the one nit, this looks good to merge.

File: src/content/en/tools/lighthouse/no-js.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+across all browsing conditions. Enhanced experiences, such as sophisticated
+styling using CSS, or interactivity using JavaScript, can be layered on top for
+the browsers that support those technologies. But basic content and page
+functionality should not rely on CSS or JavaScript.
+
+## How to pass the audit {: #how }
+
+Progressive enhancement is a large and contentious topic. If you want to strictly
+follow the strategy of progressive enhancement, then your page should be
+layered so that basic content and page functionality only requires HTML. Check
```
Comment (paullewis) [inline src/content/en/tools/lighthouse/no-js.md]: I guess this just means "should work / be viewable" before JS arrives?

**Comment by petele** [comment]
@kaycebasques - looks like this is good to go, but merges into the lighthouse branch, not next-version, can you update and assign to me please.

**Comment by kaycebasques** [comment]
@petele good to go

**Pull request merged**